docs/docs/guides/integrations/authenticated-mongodb-charts.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Authenticated MongoDB Charts
+---
+
+This integration guide shows how you can embed authenticated MongoDB Charts in your web application using ZITADEL as authentication provider.
+
+## Setup ZITADEL Application
+
+Before you can embed an authenticated chart in your application, you have to do a few configuration steps in ZITADEL Console.
+You will need to provide some information about your app. We recommend creating a new app to start from scratch.
+
+1. Navigate to your [Project](https://console.zitadel.ch/projects)
+1. Add a new application at the top of the page.
```

Comment:
```suggestion
2. Add a new application at the top of the page.
```

Reply:
in rendered markdown, numbers are incremented automatically, but sorting is easier when writing :-)